omnigibson/utils/control_utils.py

```diff
@@ -71,4 +118,7 @@ def solve(
 
         # Compute target joint positions
         ik_results = lula.compute_ik_ccd(self.kinematics, ik_target_pose, self.eef_name, self.config)
-        return np.array(ik_results.cspace_position)
+        if ik_results.success:
+            return np.array(ik_results.cspace_position)
+        else:
+            return None
```

Comment:
do we want to log a warning here? or maybe an error message? it can be `log.warn`, `log.error`, or `log.debug` so the user isn't spammed with it by default

Reply:
This is actually a very common case here - we are intentionally using the failure cases to identify times where you can't reach an object (so it's not an error failure, it's actually scenarios where no IK solution exists) - nothing to log.